### ISSUE TITLE
[Accessibility][OSX] Dropdown: Reader does not read out which item is focused

### ISSUE THREAD

**Issue by kamilakorzec** [microsoft/fluent-ui-react#1191]
## Bug Report
When navigating in open dropdown, reader does not read out currently focused item

### Steps
1. Open dropdown
2. Navigate with arrows through the dropdown items

### Expected Result
3. Screen reader correctly announces currently focused item

### Actual Result
3. Screen reader does not announce currently focused item

### Version
0.25.1

**Comment by silviuaavram** [comment]
Which Dropdown?
How to open the Dropdown?

Can you give the exact steps? Or maybe a gif with the scenario and the Voiceover output?
